### PR TITLE
Additional command to ingest docs into an Elasticsearch index

### PR DESCRIPTION
**Pull request by psybertool**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: -

## Description
Added an additional command to ingest a document into an Elasticsearch index. The command returns the response from the elasticsearch library which is already setup in the integration (no changes for that needed). If the document can't be indexed, the elasticsearch library returnes the error which is returned by the new es-index command (for example if the user has no write privileges to the given index). 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by RotemAmit — COMMENTED** [review]
Thank you very much for your contribution!
Nice work!
I saw that it is your first contribution, so welcome! You are more then welcome to join our slack community (DFIR). After joining you will be able to contact with me and other content developers (for future contributions) more easily and discuss with other developers as well. A [link](https://start.paloaltonetworks.com/join-our-slack-community) to join our community.
Here are some notes regarding to your contribution:
* Please see the following comments, and fix them.
* Please add a unit test for your new command. You can use the following guide - [link](https://xsoar.pan.dev/docs/integrations/unit-testing), as well as examples of unit tests in the HelloWorld integration and other packs from the content repository.
* Please add your new command to the Test Playbook of the integration. To do that, please upload the test-playbook (Packs/Elasticsearch/TestPlaybooks/playbook-Elasticsearch_v2_test.yml) to the XSOAR server, edit the playbook and add a task for testing your new command. When you are done - run the test-playbook to make sure it finishes successfully, if yes - download it, replace the YML of the old version of it with the new YML, and run the `demisto-sdk format` command on the YML. 
For more information about Test-Playbooks please see: https://xsoar.pan.dev/docs/integrations/test-playbooks.
For more information about the `demisto-sdk format` command please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#format.
* Please fix the [Pylint errors].(https://app.circleci.com/pipelines/github/demisto/content/304756/workflows/1087db43-ce67-48a0-a74b-a38ba29a9110/jobs/720749)

If you have any question feel free to reach out to me in slack, email, or here.

A small update, it is the holiday season in Israel, and we have a holiday in the upcoming days (24th-25th of September), so the review process might be slightly delayed, I apologize in advance.

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 1]
```diff
@@ -355,6 +355,18 @@ script:
...
       type: Number
+  - name: es-index
+    arguments:
+    - name: index
+      required: true
+      description: The name of the index to ingest into.
+    - name: doc
+      required: true
+      description: The document to be indexed.
+      type: unknown
```
[the remark this answers is not in the document]
Comment (psybertool) [reply Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:366]: Added description in yaml and readme that the document should be valid JSON format. Also changed the name to `document` (although elastic uses the word "doc" in the index itself).

**Review by RotemAmit — COMMENTED** [review]
Great work!
* The test playbook looks great but please add a task that will verify that you have received the results that you expect from the command `es-index`. More information on how to do it can be found in this [link](https://xsoar.pan.dev/docs/integrations/test-playbooks#verify-the-command-results).

**Review by RotemAmit — COMMENTED** [review]
Great work!
* Please add a unit test to the function `index_document_command`, in order to make sure that the customer receives the wanted readable_output and outputs.

Thank you very much for the fast response!

**Review by RotemAmit — COMMENTED** [review]
Great work!
I have one last comment and a suggestion: since we really appreciate your contribution, you can add a CONTRIBUTORS.json file to the root of the pack, and add your name to it. This way you will get the credit you deserve for contributing to the Elasticsearch pack. You can find more information in this [link](https://xsoar.pan.dev/docs/packs/packs-format#contributorsjson), and [here an example](https://github.com/demisto/content/blob/master/Packs/RSANetWitness_v11_1/CONTRIBUTORS.json) of a CONTRIBUTORS.json file in a different pack.
Thank you very much for your contribution!

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 2]
```diff
@@ -355,7 +355,31 @@ script:
...
+  - name: es-index
+    arguments:
+    - name: index_name
+      required: true
+      description: The name of the index to ingest into.
+    - name: document
+      required: true
+      description: The document object (JSON format) to be indexed. See Elasticsearch documentation for further information about indexing documents.
+    - name: id
+      description: The id of the indexed document (will be generated if empty).
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:367]:
```suggestion
      description: The ID of the indexed document (will be generated if empty).
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml [hunk 3]
```diff
@@ -355,7 +355,31 @@ script:
...
       type: Number
-  dockerimage: demisto/py3-tools:1.0.0.72621
+  - name: es-index
+    arguments:
+    - name: index_name
+      required: true
+      description: The name of the index to ingest into.
+    - name: document
+      required: true
+      description: The document object (JSON format) to be indexed. See Elasticsearch documentation for further information about indexing documents.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml:365]:
```suggestion
      description: The document object (JSON format) to be indexed. See Elasticsearch documentation (https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/examples.html#ex-index) for further information about indexing documents. 
```

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md [hunk 4]
```diff
@@ -178,4 +179,54 @@ Search using EQL query
...
+#### Base Command
+
+`es-index`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| index_name | The name of the index to ingest into. | Required | 
+| document | The document object (JSON format) to be indexed. See [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/examples.html#ex-index) for further information about indexing documents via the Elasticsearch client. | Required | 
+| id | The id of the indexed document (will be generated if empty). Will update the document if the defined id already exists. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md:198]:
```suggestion
| id | The ID of the indexed document (will be generated if empty). Will update the document if the defined ID already exists. | Optional | 
```
Will the document be updated or will the ID be updated?

File: Packs/Elasticsearch/ReleaseNotes/1_3_17.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Elasticsearch v2
+- Added the command **es-index** to ingest documents into an Elasticsearch index.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/ReleaseNotes/1_3_17.md:4]:
```suggestion
- Added the command ***es-index*** to ingest documents into an Elasticsearch index.
```

**Comment by ShirleyDenkberg** [comment]
@RotemAmit Doc review completed.

**Review by RotemAmit — APPROVED** [review]
Thank you very much for your contribution!

**Pull request merged**